Starter.Api/Program.cs

```diff
@@ -1,17 +1,65 @@
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Hosting;
+using Starter.Api.Requests;
+using Starter.Api.Responses;
 
-namespace Starter.Api
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddEndpointsApiExplorer();
```

Comment:
Is this necessary?

Reply:
No it's not needed to run the minimal api.
It is needed if you'd want to add Swagger to it.
But since we don't use Swagger in this project, it might as well be removed.
You're right. I'll remove it.